processing/src/main/java/org/apache/druid/segment/nested/ScalarDoubleColumn.java

```diff
@@ -39,6 +40,8 @@
 
 public class ScalarDoubleColumn implements NestedCommonFormatColumn
 {
+  private final boolean replaceWithDefault = NullHandling.replaceWithDefault();
```

Comment:
why waste a field copying a static?

Reply:
i reverted this to just have stuff using `NullHandling` again, @imply-cheddar was pushing for this, but thinking more about it I personally prefer stuff calling it directly because it sticks out more in your face that having two null handling modes is why stuff is happening in a way that feels like it gets lost with fields.